src/NServiceBus.AcceptanceTests/Performance/When_message_is_audited.cs

```diff
@@ -30,6 +30,26 @@ public class When_message_is_audited : NServiceBusAcceptanceTest
             Assert.IsTrue(context.IsMessageHandledByTheAuditEndpoint);
         }
 
+        [Test]
+        public async Task Should_contain_proper_time_sent()
```

Comment:
Guideline: Per `when_` one method attributed with `[Test]`. Does that answer the question?

So in your case

* Should_contain_processing_stats_headers
* Should_contain_proper_time_sent

should be either two `when_` or merge the asserts into one test method (if they belong together)